e2e/index.js

```diff
@@ -4,14 +4,19 @@ const wdio = require('webdriverio')
 
 const [platform = 'android'] = process.argv.slice(2)
 
-const config = require(`./config.${platform}`)
+const { configuration, options } = require(`./config.${platform}`)
 
 const test = async () => {
-  const client = await wdio.remote(config)
+  const client = await wdio.remote(configuration)
   // TODO: a high value here seems necessary to avoid starting the test
   // before the app has started. Also see the config
   await client.pause(50000)
```

Comment:
this may not actually be needed when running the web tests. I had big troubles getting the test to wait for app startup when running on mobile. the wait in the configuration (see the configuration modules) doesn't seem to work, not reliably at least, but this shouldn't be a problem in web